toolkit/committee-selection/authority-selection-inherents/src/ariadne_inherent_data_provider.rs

```diff
@@ -22,32 +23,31 @@ use {
 pub type InherentType = AuthoritySelectionInputs;
 
 #[derive(Clone, Debug, Encode, Decode)]
+/// Inherent data provider providing inputs for authority selection.
 pub struct AriadneInherentDataProvider {
+	/// Authority selection inputs.
 	pub data: Option<AuthoritySelectionInputs>,
 }
 
 #[cfg(feature = "std")]
 impl AriadneInherentDataProvider {
-	pub async fn from_mc_data(
-		candidate_data_source: &(dyn AuthoritySelectionDataSource + Send + Sync),
-		for_epoch: McEpochNumber,
-		scripts: MainChainScripts,
-	) -> Result<Self, InherentProviderCreationError> {
-		Ok(Self {
-			data: Some(
-				AuthoritySelectionInputs::from_mc_data(candidate_data_source, for_epoch, scripts)
-					.await?,
-			),
-		})
-	}
-
+	/// Creates a new [AriadneInherentDataProvider] for [sp_inherents::CreateInherentDataProviders].
+	///
+	/// Parameters:
+	/// - `client`: runtime client capable of providing [SessionValidatorManagementApi] runtime API
+	/// - `sc_slot_config`: partner chain slot configuration
+	/// - `mc_epoch_config`: main chain epoch configuration
+	/// - `parent_hash`:
+	/// - `slot`:
```

Comment:
```suggestion
	/// - `parent_hash`: parent hash of the current block
	/// - `slot`: slot of the current block
```

it's obvious but just for my ocd